scss/mixins/_grid-modifiers-vp.scss

```diff
@@ -1,6 +1,6 @@
 @each $value in $viewports {
-  $grid-mq-max: map-get($value, max );
-  $grid-mq-min: map-get($value, min );
+  $grid-mq-max: breakpoint-max(map-get($value, max));
```

Comment:
Zullen we `max` onder `min` zetten? Dat doen we ook in `baseline-loader.scss`

Reply:
done